### ISSUE TITLE
[Bug] VNC Display Scaling off

### ISSUE THREAD

**Issue by Simplepianist** [gnmyt/Nexterm#57]
### General

- [X] I have updated to the latest version of Nexterm.
- [X] My bug has not been reported yet.

### The Bug

When on remote VNC Session the mouse position inside the VNC is not where the Client desktop thinks the mouse is.
Like when going over the Desktop from left to right, you can not use the whole remote desktop cause the mouse on the client already left that window.

Feels like the scaling of the remote window is not representing the size inside the Browser

### What device are you using to access the page?

In the browser

### Which operating system is your Nexterm instance running on?

Linux

**Comment by Zixim** [comment]
[screencap-2024-09-17_10.20.27.webm](https://github.com/user-attachments/assets/da95bf7b-1351-405f-86d8-b1a282c9ce80)

Here's a screencap of the bug... as you can see there's no way to click on the start button in a windows client.
Makes remoting with VNC into a win client near unusable.

**Comment by Zixim** [comment]
Setting the browser full-screen (F11 key) prior to opening the connection mitigates the bug.

**Comment by Zixim** [comment]
@Simplepianist does this issue persist for you ? 
I can no longer reproduce :shrug:

**Comment by Simplepianist** [comment]
> @Simplepianist does this issue persist for you ? I can no longer reproduce 🤷

yeah for me it still persists event if i full-screen my browser. I would provide a video, but due to other things holding me up, i didnt get the chance to show.

Will probably deliver a screen cap by the end of next week if things get calmer.

**Comment by Zixim** [comment]
Perhaps this is browser related, i use Firefox Nightly on debian12.
An update to FF might have resolved issue for me... no idea.

**Comment by robflate** [comment]
This is still an issue. It makes VNC unusable. Tested on;

Windows: Chrome, Brave
Macos: Chrome, Safari

Is this a known bug? Can anyone else successfully use the entire screen during a VNC session?

**Comment by gnmyt** [comment]
Thanks for reporting this. The issue should be fixed with PR #372
